src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -19,7 +19,10 @@ const blankStyle = {}
 // Element weight for animations
 const heightMap = {}
 
-// Helper function to calculate element height with all opened children
+/**
+ * Helper function to calculate element height with all opened children
+ * @param {*} param0
```

Comment:
Not sure what values for `@param` to use here. What's a good code doc format to use in a Next/React js code base?

Reply:
```
@param {Object} ...
@param {string[]} activePaths - ...
@param {string} path - ...
```
You can use this as reference - https://jsdoc.app/tags-param.html

Reply:
Agree. Let's go with JSDoc. 🙂

I've updated this, let's continue this in https://github.com/iterative/dvc.org/pull/776#pullrequestreview-312201679 please.